node/src/neighborhood/mod.rs

```diff
@@ -507,6 +507,13 @@ impl Neighborhood {
     }
 
     fn handle_route_query_message(&mut self, msg: RouteQueryMessage) -> Option<RouteQueryResponse> {
+        if let Some(ref url) = msg.hostname_opt {
+            if url.contains("0.0.0.0") {
+                error!(self.logger, "Request to wildcard IP detected 0.0.0.0. Most likely because Blockchain Service URL is not set");
```

Comment:
Hmmm, I don't really like the stopper be planted in this location. I think it's already too late. We need to be vigilant to it even before we start waking the neighbourhood up. 

That's why I hope this will eventually go away with GH-799. 

Please add a TODO saying that this is provisory and refer to the aforementioned card. We'll have to make the configurator guard the way to neighbourhood and so the Node will never reach it and make it going unless the configurator finds all the required parameters set to its satisfaction.  

Another important thing: go to GH-799 and write a comment that will preserve this concern we started to have with these changes. You can mention that this should likely become another competence of the Configurator so that the blockchain service url can be checked as handy as this, in the very early stage. Mention this place in the code and also say that it definitely will need to be exchanged for broadening the competences of the Configurator. This if statement should disappear and also any linked code, like the place where you default the URL to 0.0.0.0.